ehsm_kms_service/couchdb.js

```diff
@@ -9,18 +8,31 @@ const {
   EHSM_CONFIG_COUCHDB_DB,
 } = process.env
 
-let dburl = `http://${EHSM_CONFIG_COUCHDB_USERNAME}:${EHSM_CONFIG_COUCHDB_PASSWORD}@${EHSM_CONFIG_COUCHDB_SERVER}:${EHSM_CONFIG_COUCHDB_PORT}`
-
-const nanoDb = nano(dburl)
-
 async function couchDB(server) {

```

Comment:
suggest a meaningful name like connectDB

Reply:
it will be fix